### PR TITLE
Adding StringMultiLabelProcessor

### PR DESCRIPTION
**Pull request by lastmansleeping**

### CODE REVIEW

File: python/ml4ir/base/features/feature_fns/label_processor.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import tensorflow as tf
+
+from ml4ir.base.features.feature_fns.base import BaseFeatureLayerOp
+from ml4ir.base.io.file_io import FileIO
+
+
+class StringMultiLabelProcessor(BaseFeatureLayerOp):
```
Comment (balikasg) [inline python/ml4ir/base/features/feature_fns/label_processor.py:7]: Nice code! 
- Do you plan to add a test on this? 
- Another suggestion is to improve a bit the Example in the docstring

Comment (lastmansleeping) [reply python/ml4ir/base/features/feature_fns/label_processor.py:7]: Yes, to both of those. WIP...

Comment (lastmansleeping) [reply python/ml4ir/base/features/feature_fns/label_processor.py:7]: It was a placeholder for me to go back and add an example.

**Review by mohazahran — COMMENTED** [review]
This is awesome. 

One comment I have is the order of the labels. We have no automatic way to know the order of the labels (label1-label2-label3) corresponding to the weights. But I think there is no "easy" way to set that on the top of my mind.

**Comment by lastmansleeping** [comment]
> This is awesome.
> 
> One comment I have is the order of the labels. We have no automatic way to know the order of the labels (label1-label2-label3) corresponding to the weights. But I think there is no "easy" way to set that on the top of my mind.

Replied offline on slack that we are specifying this info in the column name as a workaround for now.

**Review by mohazahran — APPROVED** [review]
LGTM

**Pull request merged**